herepy/routing_api.py

```diff
@@ -60,16 +60,16 @@ def _route(self, waypoint_a, waypoint_b, modes=None):
         route = response.response["route"]
         maneuver = route[0]["leg"][0]["maneuver"]
 
-        if any(mode in modes for mode in [RouteMode.car, RouteMode.truck]):
+        if any(str(mode) in modes for mode in [RouteMode.car, RouteMode.truck]):
```

Comment:
For each enum we have a `__str__` method which returns the str value of enum type, I guess it's better to use that.

Reply:
If you agree please update other `str(mode)` usages as well.

Reply:
This is already used according to the str() [documentation](https://docs.python.org/3/library/stdtypes.html#str):
```text
[...]If neither encoding nor errors is given, str(object) returns object.__str__()[...]
````

Reply:
Please check `RouteMode` for some of the types string value is different like `traffic_disabled = 'traffic:disabled'` it's better to use `__str(mode)__` to get the correct value.